src/utils/gameTimer.ts

```diff
@@ -0,0 +1,44 @@
+import { GameStartType, TimerType } from "..";
+
+export function frameToGameTimer(frame: number, settings: GameStartType): string {
+  const timerType = ((settings.gameInfo?.gameBitfield1 as number) & 0b11) as TimerType;
+
+  if (timerType == TimerType.DECREASING) {
+    let remainingFrames = frame > 0 ? frame : 0;
+    const startingTimerInFrames = (settings.gameInfo?.gameTimer as number) * 60;
+
+    const minutesPassed = Math.floor(remainingFrames / 60 / 60);
```

Comment:
I'm not set on this idea but you could create a couple anonymous functions for calculating minutes, seconds, and centiseconds based on frames since the logic is the same for increasing or decreasing.

Reply:
I'm not a fan of this since it would break them into multiple functions and would end up taking more space. For centiseconds, the calculation is different based on the timer type.

Reply:
1. Why is this function so complicated? Could this logic not simply be a matter of computing the duration by converting into seconds, and then formatting that using something like `moment.format("mm:ss:SS")`
2. If this function must be computed manually instead of using a library, I think would prefer two seperate functions `frameToCountdownTimer(frame: number, startingDurationFrames: number)`, `frameToCountupTimer(frame: number)`.